apps/github/client/rest_api.go

```diff
@@ -0,0 +1,92 @@
+package client
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/teamyapp/cloud/libs/errs"
+	"github.com/teamyapp/cloud/libs/telemetry"
+	"github.com/teamyapp/cloud/libs/web"
+)
+
+const (
+	gitHubAPIVersion = "2022-11-28"
+)
+
+type RestAPI struct {
+	dataCollector telemetry.DataCollector
+	httpClient    web.HTTPClient
+}
+
+func (r RestAPI) GetOrgIDByOrgName(ct context.Context, installation *Installation, orgName string) (uint64, *errs.Error) {
+	url := fmt.Sprintf("https://api.github.com/orgs/%s", orgName)
+	req, err := http.NewRequest(http.MethodGet, url, nil)
+	if err != nil {
+		internalErr := &errs.Error{
+			Code:    errs.Unknown,
+			Message: err.Error(),
+		}
```

Comment:
```suggestion
		internalErr := errs.NewError(errs.Unknown, err.Error())
```

Let's use `NewError` constructor to include stack trace in error.

Reply:
Can we make this change part of the refactor later? 
```
internalErr := errs.NewError(errs.Unknown, err.Error())
```
returns Error struct, but the log and return value expect pointers.

Reply:
I changed it back to `EmbedErr: err,` to make it consistent with the rest of the code.